### PR TITLE
mbeliaev/pep8

### PR DESCRIPTION
**Pull request by beliaev-maksim**
refactor module to follow python conventions and preserve backwards compatibility

### CODE REVIEW

File: dohq_artifactory/admin.py [hunk 1]
```diff
@@ -271,6 +272,11 @@ def encryptedPassword(self):
         Method for backwards compatibility, see property encrypted_password
         :return:
         """
+        # todo remove around April 2022
```
Comment (allburov) [inline dohq_artifactory/admin.py:275]: It costs about nothing to support these fields, let's keep it and remove TODOs

Comment (allburov) [reply dohq_artifactory/admin.py:275]: > code is strongly polluted

You haven't seen polluted code then :D 

>but we need a clean out at some moment of time

We do not. I agree with warnings, but let's remove todo. I saw warnings from 2015 in one library, and the library still support methods.

File: dohq_artifactory/admin.py [hunk 2]
```diff
@@ -271,6 +272,11 @@ def encryptedPassword(self):
...
+        warn(
```
Comment (allburov) [inline dohq_artifactory/admin.py:276]: Can we create `deprecation` function like this? https://docs.python.org/3/library/warnings.html#warnings.warn

```
def deprecation(message):
    warnings.warn(message, DeprecationWarning, stacklevel=2)
```

Comment (beliaev-maksim) [reply dohq_artifactory/admin.py]: done

File: dohq_artifactory/admin.py (hunk 1 above)
Comment (beliaev-maksim) [reply dohq_artifactory/admin.py]: > I saw warnings from 2015 in one library, and the library still support methods.
and that does not mean that we need to keep code dirty. I can probably find more examples where deprecation strategy is defined, even CPython itself :)

but for now let's remove todo and reiterate in some time, maybe next year

**Comment by beliaev-maksim** [comment]
@allburov 
I merge develop here, so pipe is fixed now

can you please review it as well?

**Pull request merged**